src/clj/puppetlabs/services/request_handler/request_handler_core.clj

```diff
@@ -262,21 +262,34 @@
       (ring-response/status http-status)
       (ring-response/content-type "text/plain")))
 
+(defn handle-request*
```

Comment:
Assuming we're going to end up calling `handle-request*` and `with-error-handling` from a PE version of `handle-request`, I wonder if it would be better to leave `handle-request*` and `with-error-handling` in "public" - since a namespace outside of this one would be referencing them.